starter-code/src/components/Contacts.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import contacts from "../contacts.json";
+
+export class Contacts extends Component {
+  state = {
+    firstFive: contacts.slice(0, 5)
+  };
+
+  addNew = () => {
+    const random = Math.floor(Math.random() * 200);
+    const randomOne = contacts.slice(random, random + 1);
+    const currContact = [...this.state.firstFive, ...randomOne];
+    this.setState({ firstFive: currContact });
+  };
+  sortByName = () => {
+    const sortMap = [...this.state.firstFive];
+    const sortedMap = sortMap.sort((a, b) => (a.name < b.name ? -1 : 1));
```

Comment:
for sorting alphabetically it's better to use `localeCompare`  because the sort method is based using numbers and lowercase and uppercase letters have different values